includes/class-wc-payments-account.php

```diff
@@ -180,9 +180,7 @@ public function get_account_status_data() {
 			];
 		}
 
-		if ( ! isset( $account['status'] )
-			|| ! isset( $account['payments_enabled'] )
-			|| ! isset( $account['deposits']['status'] ) ) {
+		if ( ! isset( $account['status'], $account['payments_enabled'] ) ) {
```

Comment:
This is probably pretty minor but using `! isset( $a, b )` is changing our original logic.
This is line is now equivalent to `! isset( $account['status'] ) && ! isset( $account['payments_enabled'] )` but previously we had `||`

Reply:
Based on this test code (props @brucealdridge), my understanding was incorrect and these are equivalent.

Reply:
Just to close the loop on this, the logic was [confirmed](https://3v4l.org/r6i6h) to behave as expected.